src/test/java/org/owasp/esapi/reference/ValidatorTest.java

```diff
@@ -297,7 +297,7 @@ public void testGetValidSafeHTML() throws Exception {
 
         assertEquals("Test. &lt;<div>load=alert()</div>", rule.getSafe("test", "Test. <<div on<script></script>load=alert()"));
         assertEquals("Test. <div>b</div>", rule.getSafe("test", "Test. <div style={xss:expression(xss)}>b</div>"));
-        assertEquals("Test.", rule.getSafe("test", "Test. <s%00cript>alert(document.cookie)</script>"));
+        assertEquals("Test.", rule.getSafe("test", "Test. <script>alert(document.cookie)</script>"));

```

Comment:
Why did you drop the null-byte test here? Like the tests under it, this is verifying that the validation works even when a special character interrupts the payload. The null-byte attack is a valid payload to verify.